### ISSUE TITLE
Make log.prefix read-only

### ISSUE THREAD

**Issue by gr2m** [hoodiehq/hoodie-client-log#15]
``` js
log = new Log('hoodie')
log.prefix // is 'hoodie'
log.prefix = 'foobar'
log.prefix // is still 'hoodie'
```

We use getters in other libraries already, like here in `hoodie-client`: https://github.com/hoodiehq/hoodie-client/blob/master/index.js#L16

---
- [x] claim this issue (@ianstalter123)
- [x] start a pull request (#22)
- [x] write tests
- [x] implement
- [x] update README
- [x] review & merge

Comment bellow, ping me in the [Hoodie Chat](http://hood.ie/chat/) or on [Twitter](https://twitter.com/gr2m/) if you have any questions

**Comment by ianstalter123** [comment]
will do this one

**Comment by gr2m** [comment]
All yours :clap:

**Comment by ianstalter123** [comment]
@gr2m the test and fix are done but should the read me be changed in an additional commit

**Comment by gr2m** [comment]
Yes, can you please remove the 

> :dog2: TO BE DONE: Make log.prefix read-only

from the README.md in an additional `docs(README): ...` commit?

**Comment by gr2m** [comment]
great work :+1: thanks!

**Comment by gr2m** [comment]
fixed via #22
